src/components/AddMovieModal/index.tsx

```diff
@@ -7,73 +8,90 @@ import {
-              <Input id="date" type="date" fullWidth />
-            </Grid>
-            <Grid item xs={6} md={8}>
-              <TextField fullWidth label="Movie URL" variant="outlined" />
-            </Grid>
-            <Grid item xs={6} md={4}>
-              <TextField fullWidth label="Rating" variant="outlined" />
-            </Grid>
-            <Grid item xs={6} md={8}>
-              <TextField
-                placeholder="genre"
-                fullWidth
-                select
-                label="Genre"
-                variant="outlined"
-              >
-                {genres.map((genre) => (
-                  <MenuItem key={genre} value={genre}>
-                    {genre.toUpperCase()}
-                  </MenuItem>
-                ))}
-              </TextField>
-            </Grid>
-            <Grid item xs={6} md={4}>
-              <TextField fullWidth label="Runtime" variant="outlined" />
-            </Grid>
-            <Grid item xs={12} md={12}>
-              <TextField
-                fullWidth
-                multiline
-                minRows={3}
-                label="Overview"
-                variant="outlined"
-                placeholder="Movie Description"
-              />
-            </Grid>
+const AddMovieModal = () => {
+  const [selectedGenres, setGenres] = useState<string[]>([]);
+
+  const handleChange = (event: SelectChangeEvent<typeof selectedGenres>) => {
+    const {
+      target: { value },
+    } = event;
+    setGenres(typeof value === "string" ? value.split(",") : value);
+  };
+
+  return (
+    <Box>
+      <Typography id="add-movie-modal-title" variant="h2">
+        Add Movie
+      </Typography>
+      <Box id="add-movie-modal-description">
+        <Grid container spacing={2}>
+          <Grid item xs={6} md={8}>
+            <TextField fullWidth label="Title" variant="outlined" />
+          </Grid>
+          <Grid item xs={6} md={4}>
+            <InputLabel htmlFor="date">Release Date</InputLabel>
+            <Input id="date" type="date" fullWidth />
+          </Grid>
+          <Grid item xs={6} md={8}>
+            <TextField fullWidth label="Movie URL" variant="outlined" />
+          </Grid>
+          <Grid item xs={6} md={4}>
+            <TextField fullWidth label="Rating" variant="outlined" />
+          </Grid>
+          <Grid item xs={6} md={8}>
+            <Select
+              multiple
+              fullWidth
+              value={selectedGenres}
+              onChange={handleChange}
+              renderValue={(selectedGenres) => {
+                if (!selectedGenres.length) return "Genres";
+                else return selectedGenres.join(", ");
+              }}
+            >
+              <MenuItem disabled key={"genres"} value="">
+                <em>Genres</em>
+              </MenuItem>
+              {genres.map((genre) => (
+                <MenuItem key={genre} value={genre}>
```

Comment:
is genre unique key?

Reply:
Yes, it is. Genres is an array of all genres, and each genre is unique.